frontend/public/components/utils/kebab.tsx

```diff
@@ -554,21 +555,23 @@ class KebabWithTranslation extends React.Component<
           'pf-m-expanded': this.state.active,
         })}
       >
-        <button
-          ref={this.dropdownElement}
-          type="button"
-          aria-expanded={this.state.active}
-          aria-haspopup="true"
-          aria-label={t('public~Actions')}
-          className="pf-c-dropdown__toggle pf-m-plain"
-          data-test-id="kebab-button"
-          disabled={isDisabled}
-          onClick={this.toggle}
-          onFocus={this.onHover}
-          onMouseEnter={this.onHover}
-        >
-          <EllipsisVIcon />
-        </button>
+        <Tooltip content={hoverMessage} isVisible={isDisabled && !!hoverMessage} trigger="manual">
```

Comment:
please make sure, this message is displayed only when the mouse cursor is over the kebab action button.